lib/kovid/helpers.rb

```diff
@@ -14,4 +16,21 @@ def dateman(date)
     date_to_parse = Date.strptime(date, '%m/%d/%y').to_s
     Date.parse(date_to_parse).strftime('%d %b, %y')
   end
+
+  # Insert commas to format numbers
+  def comma_delimit(number)
+    number.to_s.chars.to_a.reverse.each_slice(3).map(&:join).join(',').reverse
+  end
+
+  # Insert + sign to format positive numbers
+  def add_plus_sign(num)
+    num.to_i.positive? ? "+#{comma_delimit(num)}" : comma_delimit(num).to_s
+  end
+
+  # Transpose and formats cases, deaths and recovered numbers
```

Comment:
Same here with comment on code easy to decipher.

Reply:
Yes, this one is even easier to understand as the new name makes some clarifications already.